src/core/reference/src/precomp.hpp

```diff
@@ -4,14 +4,4 @@
 
 #pragma once
 
-#include <algorithm>
-#include <numeric>
-#include <stdexcept>
-#include <utility>
-#include <vector>
-
-#include <cfenv>
-#include <cmath>
-#include <cstddef>
-#include <cstdio>
-#include <cstring>
+#include "openvino/pch/precomp_core.hpp"
```

Comment:
Is possible that this file will contains OV headers for this target and standard headers can go via library linking?